jsettlers.graphics/src/main/java/jsettlers/graphics/map/draw/MapObjectDrawer.java

```diff
@@ -49,13 +50,24 @@
 import jsettlers.graphics.sequence.Sequence;
 import jsettlers.graphics.sound.SoundManager;
 
+import static jsettlers.common.movable.EMovableType.DEFAULT_HEALTH;
+
 /**
  * This class handles drawing of objects on the map.
  *
  * @author michael
  */
 public class MapObjectDrawer {
 
+	private static final int[] PASSENGER_POSITION_TO_FRONT = {2, -2, -2, 1, 1, -1, -1};
+	private static final int[] PASSENGER_POSITION_TO_RIGHT = {0, 1, -1, -1, 1, 1, -1};
+	private final int maxNumberOfPassengers = 7;
```

Comment:
You can make constants static final

or you can use PASSENGER_POSITION_TO_FRONT.length